src/customFunctions/customFetch.ts

```diff
@@ -0,0 +1,33 @@
+import { bearerTokenAuthorization } from "./../configs/bearerTokenAuthorization";
+
+type CustomFetch<TResponseJson> = {
+  fullResponse: Response;
+  json: TResponseJson;
+};
+
+export const customFetch = async <TResponseJson>(
+  input: RequestInfo,
+  init: RequestInit = {}
+): Promise<CustomFetch<TResponseJson>> => {
+  init.headers = Object.assign(bearerTokenAuthorization.headers, init.headers);
```

Comment:
nebylo by čitelnější prostě použít spread operátor, než Object.assign? ale je to to stejný, just asking